docs/topics/vs-code-extension-run-configuration.adoc

```diff
@@ -10,12 +10,12 @@ You can configure multiple run configurations to run against each project you im
 
 .Prerequisites
 
-* `mta-cli` executable installed. You can download the `mta-cli` executable from link:https://developers.redhat.com/products/mta/download[MTA download].
+* `mta-cli` executable installed. You can download the `mta-cli` executable from link:https://developers.redhat.com/products/mta/download[{ProductShortName} download].
 
 .Procedure
 
-. In *Extensions* view, click the Migration Toolkit for Applications icon (image:vs_MTA_extension_icon.png[MTA code extension]) on the Activity bar.
-. Click the *+* (plus sign) next to *Migration Toolkit for Applications* to add a run configuration.
+. In *Extensions* view, click the {ProductName} icon (image:vs_{ProductShortName}_extension_icon.png[{ProductShortName} code extension]) on the Activity bar.
```

Comment:
write the correct file name without attributes. Be sure that all names appear in the IDEs as you expect.